lib/cron/synth-switch.ts

```diff
@@ -398,8 +391,17 @@ async function main(metricsLogger: MetricsLogger) {
         };
         return formattedRow;
       });
-      metrics.putMetric(Metric.SYNTH_ORDERS, formattedResult.length, MetricLoggerUnit.Count);
-      await updateSynthSwitchRepository(configs, formattedResult, metrics);
+
+      // filter out orders that are not in the configs
+      const filteredResult = formattedResult.filter((row) => {
+        return (
+          tokenInListRaw.includes(row.tokenin.toLowerCase()) &&
+          tokenOutListRaw.includes(row.tokenout.toLowerCase())
```

Comment:
if config has two entries A->B, C->D, then tokenInList is [A, C] and tokenOutList is [B, D], this would allow C->D trade?

Reply:
ah yes good catch